tick/inference/base/learner_optim.py

```diff
@@ -153,6 +165,8 @@ def __init__(self, penalty='l2', C=1e3, solver="svrg", step=None,
         if extra_prox_kwarg is None:
             extra_prox_kwarg = {}
         self._prox_obj = self._construct_prox_obj(penalty, elastic_net_ratio,
```

Comment:
You should store `block_start` and `block_length` somewhere in the learner, such that people can retrieve their values by calling `learner.block_start`.
I suggest you do something like what we have done with `elastic_net_ratio` including the `@elastic_net_ratio.setter` which will raise a warning if one of these parameter is given when penalty is not binarsity